web/src/components/BetaBar.tsx

```diff
@@ -14,7 +14,7 @@ export const BetaBar = (): ReactElement => {
     >
       <span className="margin-left-1 margin-right-1">{Config.betaBar.betaMainText}</span>
       <PrimaryButton
-        isColor="secondary"
+        isColor="feedback"
```

Comment:
[pebble] I think this color should stay as is, when you click on an element within Figma, the color itself is identified. If we're naming the colors exactly what we are looking at in Figma it's a lot easier to figure out what prop we need to apply.
 
<img width="505" alt="Screenshot 2023-09-28 at 3 25 17 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/a7bfa570-fff8-4a61-8395-2f7d9e570a54">

Reply:
Sure